cpp/lib/include/ZoteroHarvesterConfig.h

```diff
@@ -40,6 +42,27 @@ namespace Config {
 static constexpr unsigned DEFAULT_ZEDER_ID(0);
 
 
+
+struct DownloadDelayParams {
+    unsigned default_delay_;
+    unsigned max_delay_;
+    std::map<std::string,unsigned> domain_to_default_delay_map_;
```

Comment:
If we anticipate that the number of entries in these maps could go above 20, I think we should prefer hash maps, i.e. std::unordered_map.
Also, a space is missing after the comma.